Update MPSL and SoftDevice Controller (#449)
nrfconnect/sdk-nrfxlib: mpsl/CHANGELOG.rst

All notable changes to this project are documented in this file.

Master branch
*************

Bug fixes
=========

> * Fixed an issue where clock configuration option :c:member:`skip_wait_lfclk_started` in :c:struct:`mpsl_clock_lfclk_cfg_t`
>   might have not worked as expected with nRF5340 devices (DRGN-15223).

Reviewer: ```suggestion
* Fixed an issue where the clock configuration option :c:member:`mpsl_clock_lfclk_cfg_t.skip_wait_lfclk_started` in :c:struct:`mpsl_clock_lfclk_cfg_t` did not work as expected with nRF5340 devices (DRGN-15223).
```
Updated the wording, fixed the link to the struct member (need to test if this works), and removed the line break (one sentence per line).

Reviewer: Great, but it displays the full "path" now and not only the member name - so no need to specify the struct in addition:
```suggestion
Fixed an issue where the clock configuration option :c:member:`mpsl_clock_lfclk_cfg_t.skip_wait_lfclk_started` did not work as expected with nRF5340 devices (DRGN-15223).
```

Reviewer 2: Fixed
